liiatools_pipeline/repository.py

```diff
@@ -29,20 +28,18 @@ def sync():
     https://docs.dagster.io/overview/repositories-workspaces/repositories
     """
     jobs = [
-        ssda903_clean,
-        ssda903_move_current,
-        ssda903_concatenate,
-        ssda903_move_error_report,
-        ssda903_fix_episodes,
-        ssda903_move_current_and_concat,
-        ssda903_reports,
+        clean,
```

Comment:
make sure all the jobs are here, so what's missing is:
move_error_report
fix_episodes
move_current_and_concat